alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -960,6 +883,39 @@ impl Canvas {
     }
 }
 
+/// Size of the stroke to use.
+fn calculate_stroke_size(width: usize) -> usize {
```

Comment:
```suggestion
/// Compute line width.
fn calculate_stroke_size(cell_width: usize) -> usize {
```

My main problem here is the "to use" suffix in your docs which is confusing (to use for what?). Decided to reword it completely to provide additional value over the function name itself (synonyms are more useful to figure out purpose than using the same words again).